analyzers/src/SonarAnalyzer.CSharp/Extensions/InvocationExpressionSyntaxExtensions.cs

```diff
@@ -43,5 +43,8 @@ internal static class InvocationExpressionSyntaxExtensions
 
         internal static bool IsOnBase(this InvocationExpressionSyntax invocation) =>
             (invocation.Expression as MemberAccessExpressionSyntax)?.Expression is BaseExpressionSyntax;
+
+        internal static bool IsEqualTo(this InvocationExpressionSyntax invocation1, InvocationExpressionSyntax invocation2, SemanticModel model) =>
```

Comment:
invocation vs other

Reply:
Renamed to `First` `Second`.
I tried to check what's most common in the codebase but I did not find something.